test/model.jl

```diff
@@ -123,3 +123,28 @@ end
     @test MOI.dimension(s) == 1
 
 end
+
+struct Func388 <: MOI.AbstractScalarFunction
```

Comment:
Nit: use a more descriptive name or add a link to the PR if we need to reference the discussion. 388 isn't too meaningful on its own.